app/src/main/java/com/ncnf/map/PublicEventProvider.java

```diff
@@ -0,0 +1,33 @@
+package com.ncnf.map;
+
+import com.google.firebase.firestore.GeoPoint;
+import com.ncnf.event.Event;
+import com.ncnf.event.PublicEvent;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.inject.Inject;
+
+public class PublicEventProvider {
```

Comment:
No longer needed

Reply:
This is changed in the other PR, 128/map_db